src/main/java/net/mcreator/generator/Generator.java

```diff
@@ -540,9 +556,140 @@ private List<GeneratorTemplate> getModElementGeneratorTemplatesList(ModElement e
 			}
 		}
 
+		Objects.requireNonNull(getModElementListTemplates(element, performFSTasks, generatableElement)).forEach(gtl -> {

```

Comment:
This part of the code still makes little sense to me, basically why for each list template a new list template is made? 

It would probably help understanding if there were some meaningful comments attached to the code.

Reply:
What did I forget here?

Reply:
I did not review it yet